pkg/e2e/verify/sccs.go

```diff
@@ -68,3 +129,145 @@ func checkSccPermissions(h *helper.H, clusterRole string, scc string) {
+		},
+		ObjectMeta: metav1.ObjectMeta{
+
+			Name: name,
+		},
+		Groups: []string{
+			"system:authenticated",
+		},
+
+		SELinuxContext: v1.SELinuxContextStrategyOptions{
+			Type: v1.SELinuxStrategyRunAsAny,
+		},
+		RunAsUser: v1.RunAsUserStrategyOptions{
+			Type: v1.RunAsUserStrategyRunAsAny,
+		},
+		FSGroup: v1.FSGroupStrategyOptions{
+			Type: v1.FSGroupStrategyRunAsAny,
+		},
+		SupplementalGroups: v1.SupplementalGroupsStrategyOptions{
+			Type: v1.SupplementalGroupsStrategyRunAsAny,
+		},
+	}
+	return scc
+}
+
+func createScc(scc v1.SecurityContextConstraints, h *helper.H) error {
+
+	_, err := h.Security().SecurityV1().SecurityContextConstraints().Create(context.TODO(), &scc, metav1.CreateOptions{})
+	return (err)
+}
+
+func deleteScc(scc string, h *helper.H) error {
+
+	err := h.Security().SecurityV1().SecurityContextConstraints().Delete(context.TODO(), scc, metav1.DeleteOptions{})
+	return (err)
+}
+
+//Filters pods based on namespace and label
+func filterPods(namespace string, label string, h *helper.H) (*apiv1.PodList, error) {
+
+	list, err := h.Kube().CoreV1().Pods(namespace).List(context.TODO(), metav1.ListOptions{
+		LabelSelector: label})
+	if err != nil {
+		log.Printf("Could not issue create command")
+		return list, err
+	}
+
+	return list, err
+
+}
+
+//Extracts pod names from a filtered list and counts how many are in running state
+func getPodNames(namespace string, list *apiv1.PodList, h *helper.H) ([]string, int) {
+	var notReady []apiv1.Pod
+	var ready []apiv1.Pod
+	var podNames []string
+	var total int
+	var numReady int
+	notReady = nil
+	ready = nil
+	podNames = nil
+	for _, pod := range list.Items {
+		name := pod.Name
+		podNames = append(podNames, name)
+
+		phase := pod.Status.Phase
+		if phase != apiv1.PodRunning && phase != apiv1.PodSucceeded {
+			notReady = append(notReady, pod)
+		} else {
+			ready = append(ready, pod)
+
+		}
+		total = len(list.Items)
+		numReady = (total - len(notReady))
+	}
+
+	if total != numReady {
+		log.Printf(" %v out of %v pods were/was in Ready state.", numReady, total)
+	}
+	return podNames, numReady
+}
+
+// Scales down and scales up the operator deployment to initiate a pod restart
+func restartOperator(h *helper.H, operator string, ns string) error {
+
+	log.Printf("restarting %s operator to force re-initialize pods", operator)
+
+	err := wait.PollImmediate(5*time.Second, 2*time.Minute, func() (bool, error) {
+		// scale down
+		s, err := h.Kube().AppsV1().Deployments(ns).GetScale(context.TODO(), operator, metav1.GetOptions{})
+		if err != nil {
+			return false, nil
+		}
+		sc := *s
+		sc.Spec.Replicas = 0
+		_, err = h.Kube().AppsV1().Deployments(ns).UpdateScale(context.TODO(), operator, &sc, metav1.UpdateOptions{})
+		if err != nil {
+			return false, nil
+		}
+
+		// scale up
+		s, err = h.Kube().AppsV1().Deployments(ns).GetScale(context.TODO(), operator, metav1.GetOptions{})
+		if err != nil {
+			return false, nil
+		}
+		sc = *s
```

Comment:
```suggestion
```
I have the sense that I'm probably just missing some k8s context here, but why do we need to fetch the scale again? We already have the version of it with zero replicas in the local variable `sc`. Can't we just update that one and apply it again?